helm/aws-operator/templates/rbac.yaml

```diff
@@ -63,8 +63,8 @@ rules:
       - drainerconfigs
     verbs:
       - "*"
-  # TODO drop namespaces when legacy bundles are gone and clusterapi bundle v29
-  # is the latest version.
+  # TODO drop create and delete permissions for namespaces when
+  # legacy bundles are gone and clusterapi bundle v29 is the latest version.
```

Comment:
Let's keep the TODO comment in the other helm dir in sync.

Reply:
done